CHANGELOG.md

```diff
@@ -8,5 +8,18 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - first public release
 
+### Changed
+
+- rubocop upgrade to 0.50
+- ruby upgrade to 2.2
+- bundler upgrade
+- rake upgrade
+- dkdeploy-test_environment upgrade to 2.0
+- dkdeploy-core upgrade 9.0
+- Add dependencies for `cucumber`, `rubocop`, `aruba` and `mysql2
+- upgrade vagrant box to `ubuntu/xenial64`
+- update TYPO3 8.7.8
+- moving tasks from step_definitions/typo3.rb to deploy.rb
```

Comment:
This was only for the testing environment. Please add this information to changelog.